contract_recurring_invoicing_marker/__openerp__.py

```diff
@@ -6,7 +6,7 @@
     'name': 'Markers for contract recurring invoices',
     'version': '8.0.1.0.0',
     'category': 'Contract Management',
-    'author': 'Serv. Tecnol. Avanzados - Pedro M. Baeza, '
+    'author': 'Serv. Tecnol. Avanzados - Pedro M. Baeza, bisnesmart - Gonzalo Borrás,'
```

Comment:
Keep in mind that the author key is a limited field, and I'm pretty sure it's not even the full 255 chars of the field.

I'll submit a draft change to our contribution guidelines today on this. I've been testing the rule internally, but quite a few heavy contributors seemed to like it when they saw it. Figure it's time to make the suggestion official now that it's public 😄